server/container.go

```diff
@@ -341,13 +325,38 @@ func (s *Server) createSandboxContainer(containerID string, containerName string
 		return nil, fmt.Errorf("CreateContainerRequest.ContainerConfig.Image.Image is empty")
 	}
 
-	// TODO: copy the rootfs into the bundle.
-	// Currently, utils.CreateFakeRootfs is used to populate the rootfs.
-	if err = utils.CreateFakeRootfs(containerDir, image); err != nil {
+	storageMetadata := cri.StorageRuntimeContainerMetadata{
+		Pod:           false,
+		PodName:       sb.name,
+		PodID:         sb.id,
+		ImageName:     image,
+		ContainerName: containerName,
+		MetadataName:  containerConfig.GetMetadata().GetName(),
+		Attempt:       containerConfig.GetMetadata().GetAttempt(),
+		MountLabel:    sb.mountLabel,
+	}
+	containerInfo, err := s.storage.CreateContainer(ctx, storageMetadata, containerID)
+	if err != nil {
+		return nil, err
+	}
+
+	mountPoint, err := s.storage.StartContainer(containerID)
```

Comment:
At the storage level, "Start" and "Stop" are more or less equivalent to "Mount" and "Umount".  The r/w layer is created in s.storage.CreatePodSandbox or s.storage.CreateContainer.  CreatePodSandbox also "Starts" (mounts) the infra container's layer.